index.html

```diff
@@ -2092,39 +2143,39 @@ <h3>Indirect Communication</h3>
                     <img
                         src="images/arch-high-level-application-proxy-device-alt2.png" />
                     <figcaption>High level architecture
-                        with proxy</figcaption>
+                        with Intermediary</figcaption>
                 </figure>
 
                 <p>
-                    A proxy combines <a>Exposed Thing</a> and <a>Consumed Thing</a>
+                    An <a>Intermediary</a> combines <a>Exposed Thing</a> and <a>Consumed Thing</a>
                     functionality, and relays messages that are exchanged
```

Comment:
It might be more than relaying, such as responding from a cache or transforming communication when the functionality is extended by an Intermediary.